source/indexes.txt

```diff
@@ -173,6 +173,18 @@ Limitations
 - The :guilabel:`Indexes` tab is not available if you are connected 
   to a :atlas:`Data Lake </data-lake>`. 
 
+- You can only manage Atlas Search indexes in Compass if your deployment is
+  local, has an Atlas cluster tier of M10 or larger, and runs MongoDB 7.0 
+  or higher. 
+  
+  .. tip::
+
+     For clusters running an earlier version of MongoDB or 
+     :ref:`shared tier clusters <shared-tier-cluster>`, 
+     you can manage your Atlas Search indexes from the Atlas UI, with the 
+     :ref:`Atlas CLI <atlas-cli>`, or with the 
+     :ref:`Atlas Administration API <atlas-admin-api>`. 
```

Comment:
(nit -- wording)
This is subjective and optional. Consider a slight tweak to create more parallelism. 
```suggestion
     you can manage your Atlas Search indexes using the Atlas UI, the 
     :ref:`Atlas CLI <atlas-cli>`, or the 
     :ref:`Atlas Administration API <atlas-admin-api>`. 
```

Reply:
Love parallelism, changed!